TwoNote/app/src/main/res/layout/fragment_note_detail.xml

```diff
@@ -43,11 +43,10 @@
         android:id="@+id/image_container"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintBottom_toTopOf="@id/pen_tools"
```

Comment:
probably doesn't matter, but would it make more sense to constrain this to the top of image_tools instead?